test/file04.ts

```diff
@@ -0,0 +1,22 @@
+import styled from '@emotion/styled';
+import { Theme } from '@xsite-ui/common.style';
+
+export const Container = styled.div<{ theme?: Theme }>`
+  display: grid;
+  gap: ${({ theme }) => theme?.space?.[5] || 20}px;
+  grid-template-columns: repeat(2, 1fr);
+
+  & [data-role='question'] {
+    grid-column: 1 / span 2;
+  }
+
+  & [data-role='question']:nth-of-type(4) {
+    grid-column: 1;
+  }
+
+  & [data-role='question']:nth-of-type(5) {
+    grid-column: 2;
+  }
+
+  margin-bottom: 36px;
```

Comment:
A newline is missing at the end of the file. Please add a newline at the end of the file to follow the common conventions.